tools/release-management/src/bin/version_updater.rs

```diff
@@ -0,0 +1,208 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use git_cmd::Repo;
+use time::format_description;
+use toml_edit::DocumentMut;
+
+const COMMIT_MESSAGE: &str = "chore: Update Rust & Go SDK versions";
+const PR_TITLE: &str = "Update SDK versions";
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+struct Args {
+    #[arg(short, long, default_value = "../..")]
+    root: String,
+
+    #[arg(short, long, default_value = ".")]
+    git_dir: String,
+
+    #[arg(
+        short = 'o',
+        long,
+        default_value = "coralogix/coralogix-management-sdk"
+    )]
+    github_repo: String,
+
+    #[arg(short, long, short = 'v', default_value = "0.2.1")]
+    sdk_version: String,
+
+    #[arg(short = 'c', long, default_value = "rust")]
+    rust_root: String,
+
+    #[arg(short = 'g', long, default_value = "go")]
+    go_root: String,
+}
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let args = Args::parse();
+    let cargo_toml_path = PathBuf::from(&args.root).join("rust").join("Cargo.toml");
+    let go_constants_path = PathBuf::from(&args.root).join("go").join("constants.go");
```

Comment:
Here as well, use the go_root (rust_root above)

Reply:
Done